instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import "../Login/Login.scss";
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      username: "",
+      password: "",
+      loggedIn: null
+    };
+  }
+
+  handleChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  login = e => {
+    e.preventDefault();
+    localStorage.setItem("User", this.state.username);
```

Comment:
```suggestion
    localStorage.setItem("user", this.state.username);
```
Keep this lowercase, we only want to use capitalized for specific things, like constructors